src/main/java/org/sonatype/gradle/plugins/scan/nexus/iq/scan/NexusIqPluginScanExtension.java

```diff
@@ -61,7 +61,7 @@ public NexusIqPluginScanExtension(Project project) {
     organizationId = "";
     simulationEnabled = false;
     simulatedPolicyActionId = PolicyAction.NONE.toString();
-    scanFolderPath = project.getBuildDir() + File.separator + SONATYPE_CLM_FOLDER + File.separator;
+    scanFolderPath = System.getProperty("user.dir");
```

Comment:
> Path to the project root

If that's the correct meaning, then this is the appropiated default value:

```suggestion
    scanFolderPath = project.getRootDir().getAbsolutePath();
```

Reply:
That would be ideal, but I could not find a way to detect the path to the project root.

`project.getRootDir()` is not what the developers understand as the project root i.e. the folder where the project is checked out from SCM. `project.getRootDir()` is a temporary folder where `gradle` does the build. This is not what we want. It will never contain the `.git` folder, which is needed for the commit hash discovery (the original value relied on `project.getBuildDir()`, which sometimes coincides with `project.getRootDir()` and is also a temporary folder). 

The default value I suggest above is gradle's working directory. In most cases developers run gradle commands in the root directory of the project. If that's the case, that folder contains the `.git` folder.

Reply:
> That would be ideal, but I could not find a way to detect the path to the project root.

From the [Gradle javadoc](https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html#getRootDir--):
> Returns the root directory of this project. The root directory is the project directory of the root project.

It seems `project.getRootDir()` is exactly what is needed and also it's not a temporary folder.

Reply:
You are right, @guillermo-varela. I was confused by checking the value of `project.getRootDir()` while debugging a test, which obviously runs in a temp folder.